<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarCliente</class>
 <widget class="QDialog" name="RegistrarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/elegante.jpg);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Lugar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lugar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_guardar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>Siguiente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>atras</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>REGISTRO DEL CLIENTE</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>139</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_nombre">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_apelpaterno">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_apel_materno">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_direccion">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_telefono">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>98</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Apellido Paterno</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Apellido Materno</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>direccion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Telefonos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
